src/prototype_creep_harvest.js

```diff
@@ -63,15 +63,22 @@ Creep.prototype.spawnCarry = function() {
     resourceAtPosition += _.sum(container.store);
   }
 
+  const waitTime = carryCapacity / (HARVEST_POWER * workParts) *
+    Math.sqrt(carryCapacity / Math.max(resourceAtPosition, carryCapacity));
```

Comment:
Reduce `waitTime` according to `resourceAtPosition`.  I use `Math.sqrt` in the hope that it will reach an equilibrium quickly (not call/spawn too much carries).

Reply:
K, cool, let's see